compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -57,9 +57,11 @@ import kotlinx.coroutines.withContext
 @OptIn(InternalComposeUiApi::class)
 internal abstract class BaseComposeScene(
     coroutineContext: CoroutineContext,
-    private val invalidate: () -> Unit,
+    private val invalidate: (throttledToVsync: Boolean) -> Unit,
 ) : ComposeScene {
-    protected val snapshotInvalidationTracker = SnapshotInvalidationTracker(::updateInvalidations)
+    protected val snapshotInvalidationTracker = SnapshotInvalidationTracker {
+        updateInvalidations(throttledToVsync = false)
```

Comment:
Could you check why Slider/Hover don't trigger `invalidate(throttledToVsync = false)` in `run1` with `Animation` checkbox disabled?